src/DoctrineMessageRepository.php

```diff
@@ -150,5 +150,9 @@ private function yieldMessagesForResult($stm)
                 yield $message;
             }
         }
+
+        return isset($message)
```

Comment:
Important: I broke the tests here, while working on this, but we didn't notice because the tests were not running in CI, nor did I get them to run locally (until now)